plugins/harness-feature-flags/src/components/FeatureList/FeatureList.tsx

```diff
@@ -0,0 +1,358 @@
+        return (
+          <Link href={link} target="_blank">
+            <b>{row.name}</b>
+          </Link>
+        );
+      },
+      customFilterAndSearch: (term, row: Partial<TableData>) => {
+        const temp = row?.name ?? '';
+        return temp.toLowerCase().indexOf(term.toLowerCase()) > -1;
+      },
+      customSort: (row1: Partial<TableData>, row2: Partial<TableData>) => {
+        const a = row1.name ?? '';
+        const b = row2.name ?? '';
+        return a > b ? 1 : -1;
+      },
+    },
+    {
+      title: 'Created By',
+      field: 'col2',
+      width: '30%',
+      sorting: false,
+      render: (row: Partial<TableData>) => {
+        return (
+          <Typography style={{ fontSize: 'small', color: 'grey' }}>
+            <b>{row.owner} </b>
+          </Typography>
+        );
+      },
+    },
+    {
+      title: 'Type',
+      field: 'col3',
+      width: '18%',
+      type: 'string',
+      render: (row: Partial<TableData>) => {
+        return (
+          <Typography style={{ fontSize: 'small', color: 'black' }}>
+            <b>{row.kind} </b>
+          </Typography>
+        );
+      },
+    },
+    {
+      title: 'Status',
+      field: 'col4',
+      type: 'string',
+      render: (row: Partial<TableData>) => {
+        return (
+          <Typography style={{ fontSize: 'small', color: 'green' }}>
+            <b>{row.status} </b>
+          </Typography>
+        );
+      },
+    },
+    {
+      title: 'Pipeline Configured',
+      field: 'col5',
+      type: 'string',
+      render: (row: Partial<TableData>) => {
+        return (
+          <Typography style={{ fontSize: 'small', color: 'green' }}>
+            <b>{row.pipelineConfigured} </b>
+          </Typography>
+        );
+      },
+    },
+    {
+      title: 'Created at',
+      field: 'col6',
+      type: 'date',
+      render: (row: Partial<TableData>) => {
+        const time = dayjs(row.createdAt).format('DD MMM HH:mm A');
+        return <Typography>{time}</Typography>;
+      },
+    },
+
+    {
+      title: 'Modified At',
+      field: 'col7',
+      type: 'date',
+      render: (row: Partial<TableData>) => {
+        const time = dayjs(row.modifiedAt).format('DD MMM HH:mm A');
+        return <Typography>{time}</Typography>;
+      },
+    },
+  ];
+  useAsyncRetry(async () => {
+    const query = new URLSearchParams({
+      routingId: `${accountId}`,
+      projectIdentifier: `${projectId}`,
+      environmentIdentifier: `${envId}`,
+      accountIdentifier: `${accountId}`,
+      orgIdentifier: `${orgId}`,
+    });
+    if (done) {
+      if (!envId) {
+        setEnvId(envIds[0]);
+      } else {
+        // eslint-disable-next-line no-inner-declarations
+        async function fetchFeatures() {
+          const resp2 = await fetch(
+            `${await backendBaseUrl}/harness/gateway/cf/admin/features?${query}&metrics=true&flagCounts=true&name=&summary=true`,
```

Comment:
`name` queryParam is empty on purpose?

Reply:
declared a const and passed the function, removed the no-inner-declarations